tags/environmentalsustainability.md

```diff
@@ -0,0 +1,77 @@
+# CNCF TAG Environmental Sustainability
+
+## Introduction
+
+The Paris Climate Accords outline the link between economic activity, greenhouse gas (GHG) emissions, and the impacts of climate change. GHG  emissions, primarily made from Carbon Dioxide, are released during the combustion of fossil fuels to produce electricity. To reach net zero goals, we need to identify how to reduce the environmental footprints of increasingly large data centers.
+
+## Mission Statement
+This TAG's goal is to advocate for, develop, support, and help evaluate environmental sustainability initiatives in Cloud Native Technologies.
+
+All activities of this TAG should aim to positively impact the community by changing mindset, increasing resource efficiency, and putting energy efficiency and environmental sustainability on everyone's agenda.
+
+## Responsibilities & Deliverables
+
+### Background
+We recognize that: 
+- balancing resource consumption with the desire for performance is difficult
+- there is inconsistency in how emissions are calculated and what is included by software in scopes of emission. 
```

Comment:
this sentence is not clear.

Reply:
What seems unclear to you?

Reply:
what does "software in scopes of emission" mean?

Reply:
the `scopes` refer to Scopes 1,2,3 emissions. EPA has [guidelines](https://www.epa.gov/climateleadership/scope-1-and-scope-2-inventory-guidance) on the emission factors

Reply:
Same PR, suggestion for rewording:
- there is wide inconsistency in how emissions are calculated among projects

Reply:
Agree with Dims. The original sentence is not clear. The reworking one is much better

Reply:
Like that reword @TheFoxAtWork. I will incude in @catblade PR.